### ISSUE TITLE
Problems with deterministic wallets on Windows (at least)

### ISSUE THREAD

**Issue by jonas-sk** [monero-project/monero#30]
On Windows generating and restoring of deterministic wallets won't work. The mnemonic seed differs in console and log and restoring won't work with both. Also a wallet created on linux and is working to restore on Linux, won't restore on Linux.  However, you can restore a Windows-wallet if you use the words from the log on Linux. Another thing to mention is, that in the console there are three words mssing, but both seeds have way more differences in words even after that.
Example: Log: sorrow tremble sharp plate bounce ring purpose such nothing pop outside capture neck wild thunder several endless describe valley bruise back surprise split abuse
Console: sorrow tremble sharp plate bounce ring purpose such nothing pop outside capture veral endless describe valley bruise back surprise split abuse

TL;DR: Wrapper Issue when generating on Windows and unknown issue when restoring a wallet
